subprojects/gradle/code-ownership/src/main/kotlin/com/avito/android/CodeOwnershipExtension.kt

```diff
@@ -33,4 +33,4 @@ open class CodeOwnershipExtension(
 }
 
 val ExtensionContainer.ownership: CodeOwnershipExtension
-    get() = findByType() ?: error("ownership should be available")
+    get() = findByType() ?: CodeOwnershipExtension()
```

Comment:
I consider seeing Stub instance

Reply:
`CodeOwnershipExtension` can be called a stub. It contains valid defaults by design.
Any customization is optional in this plugin.
Additional abstraction would require compatible contract. I don't see any value to split it.